src/styles/blocks/features.scss

```diff
@@ -0,0 +1,44 @@
+.features {
+  &__photo-container {
+    position: relative;
+
+    box-sizing: border-box;
+    width: 100%;
+    max-height: 320px;
+    overflow: hidden;
+
+    &::after {
+      content: "";
+      display: block;
+      padding-bottom: 100%;
+    }
+
+    @include onDesktop {
+      height: 320px;
+      padding-bottom: 0;
+    }
+  }
+
+  &__photo {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+
+    object-fit: cover;
+    object-position: center;
+
+    &:hover {
+      transition: 1s;
```

Comment:
better move transition from `:hover` to main element, because it works if when you hover on photo, but when you take back, `transition` does not work

also add `transition: [value] [duration]`